client/scss/components/_main-nav.scss

```diff
@@ -312,7 +315,6 @@ body.explorer-open {
 
         .footer-submenu {
             @include transition(max-height 0.2s ease);
-            overflow: hidden;
```

Comment:
This doesn't seem to do anything.

Reply:
Did you check that it also doesn't seem to do anything in IE? I feel like half the times when I think something doesn't do anything, it does something in IE.

Reply:
Double checked – all seems fine.